src/main/java/com/minecolonies/coremod/colony/Structures.java

```diff
@@ -119,14 +119,24 @@ private static void loadStyleMaps()
     {
         if (!Configurations.ignoreSchematicsFromJar)
         {
-            try (FileSystem fileSystem = FileSystems.newFileSystem(ColonyManager.class.getResource(SCHEMATICS_ASSET_PATH).toURI(), Collections.emptyMap()))
+            try (FileSystem fileSystem = FileSystems.getFileSystem(ColonyManager.class.getResource(SCHEMATICS_ASSET_PATH).toURI()))
             {
-                final Path basePath = fileSystem.getPath(SCHEMATICS_ASSET_PATH);
-                loadSchematicsForPrefix(basePath, SCHEMATICS_PREFIX);
+                    final Path basePath = fileSystem.getPath(SCHEMATICS_ASSET_PATH);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Make this line start at column 17. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AIndentationCheck)